lms/djangoapps/staticbook/views.py

```diff
@@ -100,7 +102,15 @@ def pdf_index(request, course_id, book_index, chapter=None, page=None):
         for entry in textbook['chapters']:
             entry['url'] = remap_static_url(entry['url'], course)
         if chapter is not None:
-            current_chapter = textbook['chapters'][int(chapter) - 1]
+            if int(chapter) <= (len(textbook['chapters'])):
+                current_chapter = textbook['chapters'][int(chapter) - 1]
+            else:
+                return redirect(
+                    reverse(
+                        pdf_index,
+                        kwargs={'course_id': course_id, 'book_index': book_index, 'chapter': 0}
```

Comment:
actually, now that I look deeper at the code, we don't want to be returning a redirect here... we just want to be setting the current chapter. I think the right move is just to add `int(chapter) <= (len(textbook['chapters']))` to the if-statement on L102. If that condition isn't satisfied, the correct default behavior is defined on L105